### DISCUSSION TITLE
Keep `EXT_mesh_gpu_instancing` when running gltfpack

### DISCUSSION THREAD

**Discussion by BoyBaykiller** [zeux/meshoptimizer#680] [General]
I am running `gltfpack -v -mi -noq -tc -i .\SimpleInstancing.gltf -o .\Compressed\Output.gltf` over [this](https://github.com/KhronosGroup/glTF-Sample-Models/tree/main/2.0/SimpleInstancing/glTF) gltf model.
It uses the EXT_mesh_gpu_instancing extension to define a bunch of instanced cubes. I can render it correctly as it, but after running it through the command the EXT_mesh_gpu_instancing extension is removed, resulting in only a single cube being rendered.

Am I missing something?

**Comment by zeux** [comment]
gltfpack currently doesn't support `EXT_mesh_gpu_instancing` in the input scenes (see https://github.com/zeux/meshoptimizer/blob/master/gltf/README.md#extensions for full list), but this is on a short list to add support for.

**Reply by BoyBaykiller** [reply]
Oh my bad for not seeing that. Thanks, and I look forward to it being supported!
